### PR TITLE
Added a simple travis build file.

### PR DESCRIPTION
**Pull request by mdevilliers**
Hopefully this will help resolve - #77 

I've cobbled this together from the instructions in the readme and the dependancies from the make file.

I think all that needs to happen to get this working is for the owner of https://travis-ci.org/mesos to switch on continuous integration for the build.

It would be ideal to add in the tests once we have this going. We would then need to make a decision whether to run the integration tests or not. 

And add the badge to the readme!

Mark

### CODE REVIEW

**Comment by fmpwizard** [comment]
@mdevilliers did you get to run this on travis? when I worked on this https://github.com/mesos/mesos-go/pull/86 , it worked up to a point, then one of the tests started failing and I run out of time to work on it. You can take a look at my PR to see the changes you need to make so you can run this project under your own github account, that way you don't have to wait for the owner of https://travis-ci.org/mesos to merge this and then see about fixing any issues that come up.

Hope it helps.

**Comment by mdevilliers** [comment]
Hey @fmpwizard  I've added in your fix for changing the the paths (really nice btw) when running as a fork.

Its running successfully under my travis [account](https://travis-ci.org/mdevilliers/mesos-go/builds/62364917).

Thanks for the help,

Mark

**Comment by jdef** [comment]
please add a readme badge

[Build Status](https://travis-ci.org/mesos/mesos-go)

**Comment by mdevilliers** [comment]
I've added the badge!

Mark

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- 1.4
+before_install: 
+- go get github.com/tools/godep
+- go get golang.org/x/tools/cmd/vet
+ #these two lines help users who fork mesos-go. It's a noop when running from the mesos organization
+- RepoName=`basename $PWD`; SrcDir=`dirname $PWD`; DestDir="`dirname $SrcDir`/mesos"
+- if [[ "$SrcDir" != "$DestDir" ]]; then mv "$SrcDir" "$DestDir"; cd ../../mesos/$RepoName; export TRAVIS_BUILD_DIR=`dirname $TRAVIS_BUILD_DIR`/$RepoName; fi
+- godep restore
+script: 
+- godep go build ./...
```
Comment (jdef) [inline .travis.yml]: why is there a godep go build AND a call to make?

**Comment by mdevilliers** [comment]
Taken from the readme ;-) 
I thought it also made sense to build the superset.

What would make sense for you?

**Comment by jdef** [comment]
How about just calling "make" and dropping the "go build". Make does it all.

**Comment by mdevilliers** [comment]
Done

**Comment by jdef** [comment]
LGTM

**Pull request merged**